### ISSUE TITLE
Expected behavior of export "Disjoint With"

### ISSUE THREAD

**Issue by dlutz2** [ontodev/robot#1144]
ROBOT 1.9.4
The export operation with a "Disjoint With" heading results in a set of class/property expressions which includes the entity being described e.g. in BFO:
"_continuant_" is described as being disjoint from "_continuant|occurrent_". 
This is the behavior of the underlying OWLAPI EntitySearcher which apparently returns the members of the mutually disjoint set including the entity of interest. 
Is including a self-disjoint value the desired result for export? Reconstructing the original ontology from the exported values (e.g. using template) requires a manual edit to remove that self-disjoint,

**Comment by matentzn** [comment]
@dlutz2 I think you are right, this does not seem desirable. Since ROBOT template associates a single ID with a set of values, the structure is different than the OWLAPI set returned by EntitySearcher.

Do you feel confident enough with OWL API to propose a fix/PR?

**Comment by dlutz2** [comment]
Yes, we're getting our act together to be able to work open source. We should be able to generate PRs for this and a couple of others.
